### ISSUE TITLE
[Request] Bring back the 2.1 assignment UI

### ISSUE THREAD

**Issue by CatsLover2006** [Mowjoh/Quasar#27]
I find that the Quasar 2.1 assignment UI is a lot easier to use than the 2.7 UI, since you can see where all your mods are relative to each other. Maybe this could be implemented in a future update?

**Comment by Mowjoh** [comment]
Hello ! 
Thanks for this feedback, I'm thinking of a solution to this.

This is a result of the way Quasar changed, for simplicity
Basically, since mods aren't scanned by default, this screen disappeared.
Because if they aren't all scanned, the screen would not show everything and that could lead to a lot of confusion.

The reason why they're not scanned by default is that too much mods wouldn't work. This new version provides a way to just transfer the mod's files as they should on the SD without needing Quasar to recognize everything, which brings compatibility with more mods.

Also, the screen you're talking about, while really useful, was too complicated. I've heard people telling me it was too complicated when they just wanted to transfer mods as is.

Still, I know it's not perfect, it was a good step towards bringing it to more people and giving them access to more mods.

As of now, I'm trying to think of a good way to show you this screen in a better way, tailored for skins, that would make this whole process of putting things in their proper slot easier. So yeah, I agree, but I'm still designing how that'll work so we keep the simplicity but still have this nice feature.

**Comment by LeMoris** [comment]
Yes, please bring this feature back! It was the only reason I used Quasar.

Also, when I try to downgrade to 2.1.0.1 (after uninstalling) I get this glitched out prompt/loading screen, and clicking anything does nothing
![bilde](https://user-images.githubusercontent.com/93150678/152362565-20939faf-c864-4161-a596-c68ff950db92.png)
